song/app.js

```diff
@@ -31,6 +31,22 @@ app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.post("/join", async (req, res, next) => {
+  const { id, name, email, profile_image, password } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      id,
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?, ?) `,
+    [id, name, email, profile_image, password]
+  );
+  res.status(200).json({ message: "userCreated" });
```

Comment:
```suggestion
  return res.status(201).json({ message: "userCreated" });
```
created라는 동작에 맞는 적절한 status code를 반환할 수 있도록 해주세요.